### ISSUE TITLE
Memo Short Code Value Parameter Not Available

### ISSUE THREAD

**Issue by zackdutra** [Triumph-Tech/Triumph-Helix#9]
### Description

The memo shortcode is missing the value parameter. 

### Actual Behavior

The example usage in the screenshot below includes a "value" parameter, but value is not actually able to be set. 

![image](https://github.com/user-attachments/assets/4985ec82-2371-4483-b258-f581b7bd80fa)

### Expected Behavior

The value parameter should be available. 

### Steps to Reproduce

- Attempt to use the shortcode exactly as shown in the example usage, copied below: 
`
{[ memo label:'memo' maxlength:'200' name:'textarea' row:'3' value:'Hello Ted!' isrequired:'true' ]}
`

- The value will not be set when the memo input is rendered. 

### Issue Confirmation

- [X] Perform a search on the [Github Issues](https://github.com/issues?q=is%3Aissue+user%3ASparkDevNetwork+-repo%3ARock) to see if your bug or enhancement is already reported.
- [X] Try to reproduce the problem on a fresh install or on the [demo site](https://rock.rocksolidchurchdemo.com/).

### Rock Version

16.6

**Comment by stanalyst** [comment]
Confirming that I am seeing this issue too.

I edited the shortcode on our system to fix it. Here is the updated lava template I'm using...

```
{% assign isRequired = isrequired | AsBoolean %}

{[ rockcontrol id:'rc-{{ '' | UniqueIdentifier }}' label:'{{ label }}' showlabel:'{{ showlabel }}' controltype:'rock-text-box' isrequired:'{{ isRequired }}' validationmessage:'{{ validationmessage }}' ]}

    <textarea name="{{ name }}" maxlength="{{ maxlength }}" rows="{{ rows }}" id="{{ id }}" class="form-control" {{ additionalattributes }} {% if isRequired %}required{% endif %}>{{ value }}</textarea>

{[ endrockcontrol ]}
```

Also added 'value' to the list of parameters for the shortcode.

**Comment by Panha-Sim** [comment]
@zackdutra This issue has been fixed in Helix v0.9, which is coming soon.
